transport_plugins/bled112/iotile_transport_bled112/bled112.py

```diff
@@ -544,6 +546,15 @@ def _process_scan_event(self, response):
         v2: There is only an advertisement and no scan response.
         """
 
+        info = None
+        # TODO make sure reading time in packets is of the same type
+        reading_time = time.time()
```

Comment:
Since we only target python 3+ now, you should always use `time.monotonic()` when looking for a numeric value that can be used to determine timeouts.  Both `time.time()` and `time.monotonic()` return `floats` in unit of seconds but `time.monotonic()` is guaranteed to never decrease whereas time.time() report utc time so can have abrupt shifts if the kernel clock is updated or changes.

Reply:
Okay, time.time() is used thought out the file

Reply:
Yea, we should probably update all of those with `time.monotonic()`

Reply:
I see, I updated those in bled112.